server.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const mongoose = require('mongoose');
+require('dotenv').config();
+
+const app = express();
+
+
+mongoose.connect(process.env.DATABASE_URL, { useNewUrlParser: true }).then(
```

Comment:
pls move to DB directory and import here if needed

Reply:
done